stable/kubecost/charts/opencost/values.yaml

```diff
@@ -0,0 +1,59 @@
+env:
+  - name: PROMETHEUS_SERVER_ENDPOINT
+    value: "http://my-prometheus-server.prometheus.svc" # The endpoint should have the form http://<service-name>.<namespace-name>.svc
+  - name: CLOUD_PROVIDER_API_KEY
+    value: "AIzaSyD29bGxmHAVEOBYtgd8sYM2gM2ekfxQX4U" # The GCP Pricing API key. This GCP api key is expected to be here and is limited to accessing google's billing API.
+  - name: CLUSTER_ID
+    value: "cluster-one" # Default cluster ID to use if cluster_id is not set in Prometheus metrics.
+
+replicaCount: 1
+
+image:
+  repository: quay.io/kubecost1/kubecost-cost-model
```

Comment:
When they stop hosting the opencost image in kubecost we should update this as well